features/search/filters.feature

```diff
@@ -79,6 +79,9 @@ Feature: Filters
     When I press "Fandoms" within "dd.exclude"
     Then the "Legend of Korra (1)" checkbox within "#exclude_fandom_tags" should not be checked
       And the "Harry Potter (1)" checkbox within "#exclude_fandom_tags" should not be checked
+    When I fill in "work_search_query" with "bad~query!!!"
+      And I press "Sort and Filter"
+    Then I should see "Your search failed because of a syntax error"
```

Comment:
This runs after the filters have been cleared, so this test passes even without your change.

I think it would be good to make a separate scenario for the bad query with an excluded tag, this scenario is already pretty long.